packages/console/src/pages/GetStarted/components/GetStartedProgress/index.module.scss

```diff
@@ -53,5 +53,19 @@
   height: 44px;
   padding: 0 _.unit(5) 0 _.unit(4);
   border-top: 1px solid var(--color-hover);
+  border-bottom: 1px solid transparent;
```

Comment:
Do not add border-bottom, as this will increase item overall height by 1px

Reply:
Maybe we can change the height to `42px`.

Reply:
updated the height to `42px` then the item height will be `44px`

Reply:
Nah there's a better way to do it 👇